src/Stratis.Sidechains.Networks/CirrusDev.cs

```diff
@@ -107,8 +107,8 @@ public CirrusDev()
             var bip9Deployments = new CirrusBIP9Deployments()
             {
                 // Deployment will go active once 75% of nodes are on 1.3.0.0 or later.
-                [CirrusBIP9Deployments.Release1320] = new BIP9DeploymentsParameters("Release1320", CirrusBIP9Deployments.FlagBitRelease1320, DateTime.Parse("2022-6-15 +0").ToUnixTimestamp() /* Activation date lower bound */, DateTime.Now.AddDays(50).ToUnixTimestamp(), BIP9DeploymentsParameters.DefaultRegTestThreshold),
```

Comment:
I see that there is no activation heights array defined for this network, is that intentional?

Reply:
Yes, it intentionally sets the activation height to 0 by omitting the array initialization.